src/Core/src/Platform/Android/ContentViewGroup.cs

```diff
@@ -97,6 +109,19 @@ protected override void OnLayout(bool changed, int left, int top, int right, int
 			var destination = _context.ToCrossPlatformRectInReferenceFrame(left, top, right, bottom);
 
 			CrossPlatformArrange(destination);
+
```

Comment:
Since the issue only occurs in the CollectionView control, it's not appropriate to apply the fix in ContentViewGroup.